README.rst

```diff
@@ -1099,6 +1099,17 @@ It is not triggered when the device is created for the first time.
 This signal is emitted when a device registers automatically through the controller
 HTTP API.
 
+``device_name_changed``
+~~~~~~~~~~~~~~~~~~~~~~~
+
+**Path**: ``openwisp_controller.config.signals.device_name_changed``
+
+**Arguments**:
+
+- ``instance``: instance of ``Device``.
+
+This signal is emitted when a device name changes.
```

Comment:
add something like "the signal is not emitted when the device is created" and please ensure this is true (with a test).